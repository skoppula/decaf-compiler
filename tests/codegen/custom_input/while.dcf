callout printf;
void main ( ) {
  int i, sum;
  int j;
  sum = 0;
  j = 0;
  while ( ( j < 10 ) ) {
    sum += j;
    j += 1;
  }  
  if ( sum != 45 ) {
    printf ( "ERROR: while loop is bad (1)\n" );
  }
  printf ( "basic while OK if no previous output\n" );
}
