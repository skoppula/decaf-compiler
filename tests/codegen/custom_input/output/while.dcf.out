basic while OK if no previous output
